packages/block-library/src/query-pagination-numbers/block.json

```diff
@@ -10,7 +10,33 @@
 	"usesContext": [ "queryId", "query" ],
 	"supports": {
 		"reusable": false,
-		"html": false
+		"html": false,
+		"color": {
+			"gradients": true,
+			"text": false
+		},
+		"typography": {
+			"fontSize": true,
+			"lineHeight": true,
+			"__experimentalFontStyle": true,
+			"__experimentalFontWeight": true,
+			"__experimentalDefaultControls": {
+				"fontSize": true
+			}
+		},
+		"spacing": {
+			"blockGap": true,
+			"__experimentalDefaultControls": {
+				"blockGap": true
+			}
+		},
+		"__experimentalLayout": {
```

Comment:
I don't think we should be adding this unless the block uses inner blocks. The current behavior of the layout block support (apply to the wrapper of blocks) is not the right one and need to be moved to the inner blocks wrapper at some point which would break this since this block doesn't have inner blocks.

Reply:
And I think we agree that the numbers should not be inner blocks?

Reply:
Yes, I think so, it's not clear what that would bring.